tests/test_math.py

```diff
@@ -257,6 +257,19 @@ def test_getitem_AnsVec(mm, dtype_):
         assert vec[i] == np_vec[i]
 
 
+@pytest.mark.parametrize("dtype_", [np.double, np.complex128])
+def test_kron_product(mm, dtype_):
+    mapdl_version = mm._mapdl.version
+    if mapdl_version < 23.2:
+        pytest.skip("Requires MAPDL 2023 R2 or later.")
+
+    m1 = mm.rand(3, 3, dtype=dtype_)
+    m2 = mm.rand(2, 2, dtype=dtype_)
+    # *kron product between matrix and another matrix
+    res = m1.kron(m2)
+    assert np.allclose(res.asarray(), np.kron(m1, m2))
+
+
```

Comment:
I think you should increase coverage by:
* Providing two vector object (so we can test line 1365-1368.
* Using a different object for m2 (so we raise some errors)